src/mlpack/core/util/backtrace.hpp

```diff
@@ -46,15 +46,20 @@ namespace mlpack {
 class Backtrace
 {
  public:
+#ifdef HAS_BFD_DL
   /**
    * Constructor initialize fields and call GetAddress to retrieve addresses
    * for each frame of backtrace.
    *
    * @param maxDepth Maximum depth of backtrace. Default 32 steps.
    */
-#ifdef HAS_BFD_DL
   Backtrace(int maxDepth = 32);
 #else
+  /**
```

Comment:
One of the stranger things I ended up doing to deal with the different signatures.

Reply:
Hehe, I guess it works though. :)